vault/logical_system_paths.go

```diff
@@ -410,9 +436,37 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 
 			Fields: map[string]*framework.FieldSchema{},
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ReadOperation:   b.handleRekeyRetrieveRecovery,
-				logical.DeleteOperation: b.handleRekeyDeleteRecovery,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Callback: b.handleRekeyRetrieveRecovery,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"nonce": {
+									Type:     framework.TypeString,
+									Required: true,
+								},
+								"keys": {
+									Type:     framework.TypeMap,
+									Required: true,
+								},
+								"keys_base64": {
+									Type:     framework.TypeMap,
+									Required: true,
+								},
+							},
+						}},
+					},
+				},
+				logical.DeleteOperation: &framework.PathOperation{
+					Callback: b.handleRekeyDeleteRecovery,
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
```

Comment:
```suggestion
						http. StatusNoContent: {{
```

deletes call the same function/ returns no content
https://github.com/hashicorp/vault/blob/4b8bd15e28ad3194159e7601fe76a994688988d4/vault/logical_system.go#L909